superset-frontend/spec/javascripts/components/ColumnOption_spec.jsx

```diff
@@ -19,9 +19,9 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import ColumnOption from 'src/components/ColumnOption';
-import ColumnTypeLabel from 'src/components/ColumnTypeLabel';
-import InfoTooltipWithTrigger from 'src/components/InfoTooltipWithTrigger';
+import { InfoTooltipWithTrigger } from '@superset-ui/control-utils';
+import ColumnOption from '../../../src/components/ColumnOption';
+import ColumnTypeLabel from '../../../src/components/ColumnTypeLabel';
```

Comment:
These two paths should use the absolute path, 'src/components/ColumnOption' and 'src/components/ColumnTypeLabel' based on [this PR](https://github.com/apache/incubator-superset/pull/9761/). I thought the linter might even catch it, but perhaps not.

Reply:
I think I commited the wrongs stuff here when i rebased. oopsies